DocumentsFromSnapshotMigration/src/test/java/org/opensearch/migrations/bulkload/SourceTestBase.java

```diff
@@ -269,7 +274,11 @@ public static DocumentsRunner.CompletionStatus migrateDocumentsWithOneWorker(
                 return reader;
             });
 
-            var defaultDocTransformer = new TransformationLoader().getTransformerFactoryLoader(RfsMigrateDocuments.DEFAULT_DOCUMENT_TRANSFORMATION_CONFIG);
+
+            var defaultDocTransformer = new TransformationLoader().getTransformerFactoryLoader(
```

Comment:
minor, but we don't have to name it default now that it's specified as a function arg